src/coresyftools/coresyftools/tests/test_tool.py

```diff
@@ -312,3 +312,39 @@ def test_can_use_custom_manifest_name(self):
         self.assertEqual(tool.manifest['name'], 'MyFairTool')
         os.remove(manifest_file_name)
 
+    def test_zip_output(self):
+        """Test the method to compress an output folder"""
+        tool = CoReSyFTool(self.runfile, 'manifest.json')
+        tool.write_to_zipfile('coresyftools/tests/tool6/myoutput/')
+        files_ = os.listdir('coresyftools/tests/tool6')
+        assert 'myoutput.zip' in files_
+        archive = ZipFile('coresyftools/tests/tool6/myoutput.zip', 'r')
+        self.assertGreater(len(archive.namelist()), 0)
```

Comment:
This almost enough. But to be completely sure I would test if the specific files which are supposed to be in the zip are there.